distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -307,6 +307,32 @@ type Not_Invokable
     to_display_text : Text
     to_display_text self = "Type error: expected a function, but got "+self.target.to_display_text+"."
 
+@Builtin_Type
+type Private_Access
+    ## PRIVATE
```

Comment:
No, it shouldn't be `## PRIVATE`, it should be `private`!

Reply:
Good point. But let's do all this refactoring in https://github.com/enso-org/enso/issues/9762. This PR is already big enough, so let's only test it here, and let's leave the refactoring of the whole stdlib codebase for a follow-up PR.